mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -613,6 +618,19 @@ The `web_server_network_access_control` supports:
   (Required)
   `cidr_block` must be specified in CIDR notation.
 
+<a name="nested_data_retention_config"></a>The `data_retention_config` block supports:
+* `task_logs_retention_config` - 
+  (Optional)
+  The configuration setting for Task Logs.
+  [documented below](#nested_task_logs_retention_config)
+
+<a name="nested_task_logs_retention_config"></a>The `task_logs_retention_config` block supports:
+* `storage_mode` - 
+  (Optional)
+  The mode of storage for Airflow workers task logs. Values for storage mode are 
+  `CLOUD_LOGGING_ONLY` for enabling logs in cloud logging only feature and 
+  `CLOUD_LOGGING_AND_CLOUD_STORAGE` for disabling logs in cloud logging only feature.
```

Comment:
Generally we try to match the API documentation for features like this; I don't see these fields in the current public docs, so I can't confirm whether this matches the public docs.

However, I think this will be confusing to users as written - although this is a new "cloud logging only" feature that's getting launched, from the Terraform user perspective, this is a "storage mode" feature with two options: CLOUD_LOGGING_ONLY and CLOUD_LOGGING_AND_CLOUD_STORAGE. I could see it potentially having additional options in the future like CLOUD_STORAGE_ONLY, for example.

What would you think about something like:

```suggestion
  The mode of storage for Airflow workers task logs. Values for storage mode are 
  `CLOUD_LOGGING_ONLY` to only store logs in cloud logging and 
  `CLOUD_LOGGING_AND_CLOUD_STORAGE`  to store logs in cloud logging and cloud storage.
```
Or alternatively:

```suggestion
  The mode of storage for Airflow workers task logs. Values for storage mode are 
  `CLOUD_LOGGING_ONLY` or `CLOUD_LOGGING_AND_CLOUD_STORAGE`.
```

Reply:
Agreed.